openstf/model_selection/model_selection.py

```diff
@@ -5,17 +5,18 @@
 from datetime import timedelta
 
 import numpy as np
+import pandas as pd
 import structlog
 
 
 def split_data_train_validation_test(
-    data,
-    test_fraction=0.0,
-    validation_fraction=0.15,
-    back_test=False,
-    period_sampling=True,
-    period_timedelta=timedelta(days=2),
-):
+    data: pd.DataFrame,
+    test_fraction: float = 0.0,
+    validation_fraction: float = 0.15,
+    back_test: bool = False,
+    period_sampling: bool = True,
+    period_timedelta: timedelta = timedelta(days=2),
```

Comment:
It might be unsafe to use a timedelta instance here (similar to not use [] or {} as default arguments). Perhpas better to just use a plain datatype in days/hours here. Especially since we are not using any of the timedelta features -> `int(period_timedelta / delta)`. I'm not even sure what happens here? Which unit will we get the timedelta? Seconds, days etc?